### PR TITLE
Bazel updates for fuel tools5

### PR DESCRIPTION
**Pull request by JShep1**

### CODE REVIEW

File: BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+IGN_DESIGNATION = "/tmp"
+
+CMAKE_INSTALL_PREFIX = "/tmp"
+
+CMAKE_INSTALL_DATAROOTDIR = "/tmp"
+
+library_location = "/usr"
+
+IGN_LIBRARY_PATH = "/usr"
+
```
Comment (mahiuchun) [inline BUILD.bazel:38]: Could we add 

```
licenses(["notice"])

exports_files(["LICENSE"])
``` 

around here?

File: BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    deps = [
+        "//ign_common",
+        "//ign_msgs",
+        "@json//:headers",
+    ],
+)
+
+cc_library(
+    name = "ign_fuel_tools",
+    srcs = ["libignition-fuel-tools5.so"],
```
Comment (mahiuchun) [inline BUILD.bazel:148]: could we add real sources here and let the .so binary target depend on this library target?

File: BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+cc_library(
+    name = "ign_fuel_tools",
+    srcs = sources + private_headers + public_headers,
+    includes = ["include"],
+    deps = [
+        IGNITION_ROOT + "ign_common",
+        IGNITION_ROOT + "ign_msgs",
+        IGNITION_ROOT + "ign_bazel:utilities",
+        "@curl",
```
Comment (mahiuchun) [inline BUILD.bazel:108]: these are not yet defined in `ign-bazel/repositories.bzl`?

Comment (mjcarroll) [reply BUILD.bazel:108]: I had noticed that these were still using `linkopts` as I was cleaning up some other things, here is where they are added: https://github.com/ignitionrobotics/ign-bazel/pull/32

**Pull request closed without merging**